src/OrchardCore/OrchardCore.DisplayManagement.Liquid/LiquidViewsFeatureProvider.cs

```diff
@@ -35,7 +36,8 @@ public LiquidViewsFeatureProvider(IOptions<TemplateOptions> templateOptions)
                     _sharedPaths = new List<string>();
 
                     var filePaths = templateOptions.Value.FileProvider.GetViewFilePaths(
-                        Application.ModulesPath, new[] { LiquidViewTemplate.ViewExtension },
+                        Application.ModulesPath,
+                        new[] { LiquidViewTemplate.ViewExtension },
```

Comment:
We could move this into a `static` field

Reply:
Okay, I will do it